lib/index.js

```diff
@@ -359,80 +217,111 @@ exports.createFunction = async (ctx, functionSpecification) => {
         }
 
         if (response.status === 204 || (response.body && response.body.status === 'success')) {
-            return;
-        } else {
-            throw response.body;
+            if (response.body && response.body.location) {
+                return response.body.location;
+            }
+            else {
+                response = await Superagent.get(
+                    `${ctx.body.baseUrl}/v1/account/${ctx.body.accountId}/subscription/${ctx.body.subscriptionId}/boundary/${ctx.body.boundaryId}/function/${ctx.body.functionId}/location`
```

Comment:
Low priority: pull the base function out so that all of these superagent calls have a much clearer url component?